euphony/src/main/cpp/core/source/UTF16Charset.cpp

```diff
@@ -0,0 +1,57 @@
+#include "UTF16Charset.h"
+#include <bitset>
+#include <codecvt>
+#include <iomanip>
+
+using namespace Euphony;
+
+const unsigned int BIT_COUNT_IN_HEX_NUM = 4;
+const unsigned int BYTE_COUNT = 2;
+const unsigned int HEX_NUM_COUNT = BYTE_COUNT * 8 / BIT_COUNT_IN_HEX_NUM;
```

Comment:
These 3 constant declarations are used only in UTF16Charset.
how about declaring them as a private constant member in the UTF16Charset class?